examples/gatsby/gatsby-spa/src/pages/ExperienceSlugExample.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { ExperienceRoot, useFetchBySlug } from '@contentful/experiences-sdk-react';
+import { useContentfulClient } from '../utils/useContentfulClient';
+
+const slug = 'ExperienceSlugExample'
+const localeCode = 'en-US';
```

Comment:
I would love to not hardcode these, and instead use dynamic routing like we do in nextjs so you can have multiple experiences rendering this same page template. Is this something I could possibly contribute to the branch/project?